lib/Semantics/mod-file.cpp

```diff
@@ -186,7 +193,7 @@ void ModFileWriter::PutSymbols(const Scope &scope) {
 // Emit a symbol to decls_, except for bindings in a derived type (type-bound
 // procedures, type-bound generics, final procedures) which go to typeBindings.
 void ModFileWriter::PutSymbol(
-    std::stringstream &typeBindings, const Symbol &symbol) {
+    llvm::raw_ostream &typeBindings, const Symbol &symbol) {
```

Comment:
raw_string_ostream?

Reply:
We've decided to replace all std::stringstream  parameters  by llvm::raw_ostream ones.

Reply:
+1 for improved flexibility.